scripts/nm-vpn

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 
 CONNECTION=$(nmcli -t connection show --active | awk -F ':' '
-/tun0/{vpn="ON"} /vpn/{name=$1}
+/wg0|tun0/{vpn="ON"} /wireguard|tun0|vpn/{name=$1}
 END{if(vpn) printf("%s", name)}')
```

Comment:
Try this instead:
```
CONNECTION=$(nmcli -t connection show --active | awk -F ':' '
/wg0|tun0/{vpn="ON"}; {if(/wireguard|vpn/) {vpn_name=$1} else if (/tun0/){tun_name=$1}}
END{if ( (vpn) && (vpn_name)) {printf("%s", vpn_name)}  else if( (vpn) && (tun_name)) {printf("%s", tun_name)}}')
```
This will prioritise `wireguard` and `vpn` names and fall back to `tun0` names.
Quite verbose, but performant enough and it'll make it easier to read for others.

Let me know if it works with your VPN client.